ChangeLog

```diff
@@ -2,6 +2,14 @@ mbed TLS ChangeLog (Sorted per branch, date)
 
 = mbed TLS x.x.x branch released xxxx-xx-xx
 
+API Changes
+   * Extend the platform module with an abstraction mbedtls_platform_gmtime()
+     whose implementation should behave as a thread safe version of gmtime().
+     This allows users to configure such an implementation at compile time when
+     the target system cannot be deduced automatically. At this stage Mbed TLS
+     is only able to configure implementations for Windows and POSIX C
```

Comment:
Minor: I think it would be preferable to say "automatically select" instead of "configure", as otherwise some people might get the wrong impression that you can't set your own function on other platforms, which you obviously can.